easybuild/easyconfigs/p/PGI/PGI-16.1-GCC-4.9.2-2.25.eb

```diff
@@ -0,0 +1,26 @@
+name = 'PGI'
+version = '16.1'
+
+homepage = 'http://www.pgroup.com/'
+description = "C, C++ and Fortran compilers from The Portland Group - PGI"
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+sources = ['pgicdk-%(version_major)s%(version_minor)s.tar.gz']
```

Comment:
I'd recommend NOT installing cdk. The only difference as far as I can tell is that it ships MVAPICH and MPICH with it. See https://www.pgroup.com/doc/pgirn.pdf, section 1.2.1:

> #### 1.2.1. Additional Components for PGI CDK
> The PGI CDK for Linux includes additional components available for download from the PGI
> website, but not contained in the installation package:
> ‣ MVAPICH2 MPI libraries, version 2.0, available for 64-bit development environments.
> ‣ MPICH MPI libraries, version 3.2, available for 64-bit development environments.